logger/logger_test.go

```diff
@@ -125,7 +122,6 @@ func TestLogger_LogWithOptions(t *testing.T) {
 		assert.Equal(t, "anotherapp", line["app"])
 		assert.Equal(t, "production", line["env"])
 		assert.Equal(t, "error", line["level"])
-		assert.Equal(t, now.UnixNano()/int64(time.Millisecond), int64(line["timestamp"].(float64)))
```

Comment:
why are we removing this?

Reply:
value is going to differ run to run and so we decided to assert on the existence rather than the value of the timestamp (see below)

Reply:
not sure I follow. existing code was storing the `now` value so that we _could_ assert on the exact value (that was done in #13). it should not be different per run when we are supplying the value in options - if it is, that's a bug.

Reply:
![Uploading Screen Shot 2020-09-28 at 10.48.32 AM.png…]()

Reply:
seems like a regression/bug then, right? this test passed before, it does not now.

Reply:
resolved this issue